@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -2018,4 +2038,20 @@ export class TextElementsRenderer {
         this.m_overloaded = newOverloaded;
         return this.m_overloaded;
     }
+
+    /**
+     * Project point to screen and check if it is on screen or within a fixed distance to the border.
+     *
+     * @param point center point of label.
+     * @param outPoint projected screen point of label.
+     */
+    private labelPotentiallyVisible(point: Vector3Like, outPoint: THREE.Vector2): boolean {
+        const projectionResult = this.m_screenProjector.projectAreaToScreen(
+            point,
+            MAX_HORIZONTAL_DISTANCE_TO_BORDER * 2 * this.m_pixelToNDC,
+            MAX_VERTICAL_DISTANCE_TO_BORDER * 2 * this.m_pixelToNDC,
```

Comment:
This conversion to NDC can be done in ScreenProjector, it already has the screen size in pixels.